modules/ivc_champva/app/form_mappings/vha_10_7959c.json.erb

```diff
@@ -43,16 +43,16 @@
   "form1[0].#subform[0].#area[8].CheckBox1-MedicaidSA-2[0]":                  "<%= form.data.dig('applicant_secondary_insurance_type') == 'medicaid' ? 1 : 0%>",
   "form1[0].#subform[0].#area[8].CheckBox-RxDiscount-2[0]":                   "<%= form.data.dig('applicant_secondary_insurance_type') == 'rx_discount' ? 1 : 0%>",
   "form1[0].#subform[0].#area[8].#area[9].CheckBox-Medigap-2[0]":             "<%= form.data.dig('applicant_secondary_insurance_type') == 'medigap' ? 1 : 0%>",
-  "form1[0].#subform[0].#area[8].#area[9].DropDownList-Medigap-2[0]":         "<%= form.data.dig('secondary_medigap_plan') %>", 
-  "form1[0].#subform[0].#area[8].CheckBox-Other-2[0]":                        "<%= form.data.dig('applicant_secondary_insurance_type') == 'other' ? 1 : 0%>", 
+  "form1[0].#subform[0].#area[8].#area[9].DropDownList-Medigap-2[0]":         "<%= form.data.dig('secondary_medigap_plan') %>",
+  "form1[0].#subform[0].#area[8].CheckBox-Other-2[0]":                        "<%= form.data.dig('applicant_secondary_insurance_type') == 'other' ? 1 : 0%>",
   "form1[0].#subform[0].Comments-Ins-2[0]":                                   "<%= form.data.dig('secondary_additional_comments' ) %>",
   "form1[0].#subform[0].NameofInsurance-2[0]":                                "<%= form.data.dig('applicant_secondary_provider') %>",
   "form1[0].#subform[0].Date-NameInsurance-2[0]":                             "<%= form.data.dig('applicant_secondary_effective_date') %>",
-  "form1[0].#subform[0].Date-TermnNameInsurance-2[0]":                        "<%= form.data.dig('applicant_secondary_expiration_date') %>",  
+  "form1[0].#subform[0].Date-TermnNameInsurance-2[0]":                        "<%= form.data.dig('applicant_secondary_expiration_date') %>",
   "form1[0].#subform[0].RadioButtonList[3]":                                  "<%= form.data.dig('applicant_secondary_through_employer') ? 0 : 1 %>",
   "form1[0].#subform[0].RadioButtonList[4]":                                  "<%= form.data.dig('applicant_secondary_eob') == 'hasEob' ? 0 : 1 %>",
   "form1[0].#subform[0].RadioButtonList[5]":                                  "<%= form.data.dig('applicant_secondary_has_prescription') ? 0 : 1 %>",
-  "form1[0].#subform[0].Signature[0]":                                        "<%= form.data['signature'] %>",
+  "form1[0].#subform[0].Signature[0]":                                        "<%= form.data['statement_of_truth_signature'] %>",
   "form1[0].#subform[0].DateTimeField1[0]":                                   "<%= form.data.dig('certification_date') %>",
```

Comment:
@michaelclement is going to normalize `statement_of_truth_signature` in vets-website